app/unitPages.rb

```diff
@@ -156,7 +156,16 @@ def getNavBar(unit, navItems, level=1)
 def getPageBreadcrumb(unit, pageName, issue=nil)
   ((!pageName and !issue) || pageName == "home") and return []
   if issue
-   return [{name: "Volume #{issue[:volume]}, Issue #{issue[:issue]}",
+   vol = "Volume #{issue[:volume]}"
+   iss = "Issue #{issue[:issue]}"
+   if !issue[:numbering]
+     name = vol + ", " + iss 
+   elsif issue[:numbering] == "volume_only"
+     name = vol 
+   else
+     name = iss 
+   end
```

Comment:
Duplicative code to what is in JournalLayout.jsx. But since Breadcrumb is generated on Ruby side, this was the easiest approach.